### ISSUE TITLE
Allow to run custom commands in cross containers to be able to use strip after the build

### ISSUE THREAD

**Issue by lumasepa** [cross-rs/cross#1266]
### Checklist

- [X] I've looked through the [issues and pull requests](https://github.com/cross-rs/cross/issues?q=) for similar request
- [ ] This feature could be solved with a [custom docker image](https://github.com/cross-rs/cross#custom-docker-images) (optional)

### Describe your request

I think will be really useful to have a cross subcommand to run a custom command in the docker container. It could be something like `cross runsh --target <target> "shell commands"`

### Describe why this would be a good inclusion for `cross`

In my use case, I want to strip the binaries and I need the target strip to do it properly.
I know that you can configure striping in `Cargo.toml` but in Apple targets don't work, it shows the next error (this should be solved in cargo anyway probably is using the host strip to strip the target). 
```
warning: stripping debug info with `strip` failed: exit status: 1
  |
  = note: strip: /target/aarch64-apple-darwin/release/deps/run_local_network-899c3ee2eced8839: file format not recognized
```

If I we have the option to run custom commands in cross I could run `cross runsh --target <target> "<target>-strip path/to/my/bin"`. I know that I can use the docker container directly but is handy to have cross setting up the volumes and all needed environment.

Also I can see how others can beneficiate from this if they want to run other target specific tools like objdump or nm in their binaries/libs.

**Comment by lumasepa** [comment]
I will be happy to implement it if the team agrees that is a good use case and we can include it in cross.

**Comment by NobodyXu** [comment]
I think this is a duplicate of #817

**Comment by Emilgardis** [comment]
I think it's in the same area as #716 but not the same thing. This one is also much easier to implement.

I think this would be good to have also for debugging issues in a cross container

**Comment by Emilgardis** [comment]
Implementing this should be pretty easy, it'd have to be implemented as a `cross-util` command and would just have to modify [`docker::run`](https://github.com/cross-rs/cross/blob/b7f7ace6681511de8df3bfbfd05652e369588875/src/docker/mod.rs#L44) to allow for an interpretive session

**Comment by Emilgardis** [comment]
implemented in #1280 🎉
